README.md

```diff
@@ -517,16 +517,20 @@ Now we'll sign the RPM:
 
 ```
 rpm --resign <rpm>.rpm  # --addsign would allow us to apply multiple signatures to the RPM
-rpm -qi<file.rpm>  # should now show that the file is signed
+rpm -qi <file.rpm>  # should now show that the file is signed
 rpm -Kv  # should contain NOKEY errors in the lines containing Signature
 # This is because the the (public) key of the RPM signing key is not present,
 # and must be added to the RPM client config to verify the signature:
-sudo rpm --import <publicKey>.asc
+sudo rpm --import <publicKey>.asc (keys are in the lfs repos)
 rpm -Kv  # Signature lines will now contain OK instead of NOKEY
+rpm qi <file.rpm>  # Copy the KEY ID so you can paste it into the next command
```

Comment:
nit: can the key ID be extracted via bash instead of copy-pasting?

Reply:
looking into it...

Reply:
Unfortunately it doesn't look like there's a direct rpm query to get just the key ID, but we could do something like `rpm -Kv <file.rpm> | grep -o "key ID .*:" | grep -Ei '[[:alnum:]]{8}*'` to get the key ID (gonna see if there's a better way)

Reply:
Okay I think `rpm -Kv <file.rpm> | grep -oP 'key ID \K\w+'` works well... will update